src/scss/components/table/_index.scss

```diff
@@ -48,10 +48,10 @@
 }
 
 // ===== States =====
-@state-selected-bg: @cyan-50;
+$state-selected-bg: $cyan-50;
 
 // ===== Tables =====
-@table-border-color: @gray-300;
+$table-border-color: $gray-300;
 
 /* ========== Tables ========== */
 
```

Comment:
Please convert layout comment [Line 58-68] to use triple slash (`///`) syntax for documentation.

Reply:
Done the changes.